fx_crash_sig/symbolicate.py

```diff
@@ -143,6 +143,10 @@ def __get_symbolicated_trace(self, sym_request):
             "User-Agent": f"fx-crash-sig/{__version__}",
         }
         response = requests.post(self.api_url, headers=headers, json=sym_request)
+
+        if response.status_code == 400 and self.verbose:
+            print(f"fx-crash-sig: BadResponse: {response.text}", file=sys.stderr)
```

Comment:
Previously, it would hit the `raise_for_status()` and then we'd lose the response data forever so it's hard to know what was bad about the request.